pep-0657.rst

```diff
@@ -294,27 +297,37 @@ Will be displayed as::
             ^^^
     ZeroDivisionError: division by zero
 
+Maintaining the current behavior, only a single line will be displayed
+in tracebacks. For instructions that span multiple lines (the end offset
+and the start offset belong to different lines), the end line number must
+be inspected to know if the end offset applies to the same line as the
+starting offset.
 
 Opt-out mechanism
 ^^^^^^^^^^^^^^^^^
 
-To offer an opt-out mechanism for those users that care about the storage and
-memory overhead, the functionality will be deactivated along with the extra
-information when Python is executed in "opt-2" optimized mode (``python -OO``)
-resulting in ``pyc`` files not having the overhead associated with the extra
-required data.
+To offer an opt-out mechanism for those users that care about the
+storage and memory overhead and to allow third party tools and other
+programs that are currently parsing tracebacks to catch up the following
+methods will be provided to deactivate this feature:
+
+* A new environment variable: `PYDEACTIVATEDEBUGRANGES`
+* A new command line option for the dev mode: `python -Xnodebugranges`.
```

Comment:
Why not align the names better? E.g. `PYNODEBUGRANGES`.

Reply:
Good point!